README.md

```diff
@@ -66,6 +68,36 @@ may need to change the permissions manually for the generated files after the
 These commands are the ones used by our CI system to ensure that the Protobuf
 files stay semantically valid.
 
+## Build Using Bazel
+
+![build proto](https://github.com/p4lang/p4runtime/workflows/build%20proto/badge.svg)
+
+The protobufs can also be built using [Bazel](https://bazel.build/).
+The Bazel WORKSPACE and BUILD files are located in the [proto folder](proto/).
+
+To build, run
+```sh
+cd proto && bazel build //...
+```
+
+We run [continuous integration](.github/workflows/ci-build-proto.yml) to ensure
+this works with the latest version of Bazel.
+
+To depend on P4Runtime in your own Bazel project, add the following to your
+WORKSAPCE file:
+```bazel
+git_repository(
+  name = "com_github_p4lang_p4runtime",
+  url = "https://github.com/p4lang/p4runtime.git",
+  strip_prefix = "proto",  # The proto folder is the root of the Bazel workspace.
+  ..
+)
+load("@com_github_p4lang_p4runtime//:p4runtime_deps.bzl", "p4runtime_deps")
+p4runtime_deps()
+```
+You will likely also have to load transitive dependencies. You can see how to
+do so in [p4runtime's WORKSPACE file](proto/WORKSPACE.proto).
```

Comment:
Is this the wrong path? .proto seems wrong.

Reply:
Good catch; fixed.
(I wish GitHub would automatically check broken links)